### DISCUSSION TITLE
Antialiasing

### DISCUSSION THREAD

**Discussion by mbtemiz** [hjoykim/THREE#16] [Q&A] [answered]
Hi,
First of thank you for this project. I couldnt find an option for antialiasing neither in the examples nor in the project files - if I am not missing anything. How do we turn on antialiasing?

**Comment by hjoykim** [comment, marked as answer]
Hello

if OpenTK.GLControl was created without antialias option in Winform,  it's impossible to change glControls's antialias property , 
if you want to set antialias you must destory glControl and recreate by 

glControl.Dispose();
GraphicsMode mode = new GraphicsMode(new ColorFormat(32),16, 0, 8);
glControl = new GLControl(mode)
// and reassign all of glControl property and method
in Example.cs

I've tried before, but for some unknown reason
The build was not successful.

Let's take a look again and I'll commit source and example

**Reply by mbtemiz** [reply]
Thank you for the answer.
By the way the following graphicsmode configuration gave me a better result;
glControl = new OpenTK.GLControl(new OpenTK.Graphics.GraphicsMode(32, 24, 0, 8));

**Comment by hjoykim** [comment]
thanks your interests and good alternative comment
